src/server.tsx

```diff
@@ -378,6 +382,28 @@ app.post('/epic/compare-variant-decision', async (req: express.Request, res: exp
     res.send('thanks');
 });
 
+app.get(
+    '/amp/experiments_data',
+    cors({
```

Comment:
The `/amp/experiments_data` endpoint is solely for the DCR backend right?
In that case this `cors` block can be removed

Reply:
👍🏼